stanford/roles/deployment/tasks/main.yml

```diff
@@ -60,6 +78,7 @@
       Name: "{{ COMMON_DEPLOYMENT }}-route-public-{{ DEPLOYMENT_NUMBER }}"
       Number: "{{ DEPLOYMENT_NUMBER }}"
   register: deployment_route_public
+  when: DEPLOYMENT_SUBNET_PUBLIC_IDS is not defined
```

Comment:
We don't need to update the public route table if we're not updating the public subnet.